File: Estrutura_Condicional/Java/Ex1061/Main.java
package Ex1061;

import java.util.Scanner;
import java.time.LocalDateTime;
import java.time.Duration;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        String[] entradaInicio = scanner.nextLine().split(" ");
        int entradaInicioDia = Integer.parseInt(entradaInicio[1]);
        String[] entradaInicioTempo = scanner.nextLine().split(" : ");
        LocalDateTime inicio = LocalDateTime.of(2024, 4, entradaInicioDia,
                                                  Integer.parseInt(entradaInicioTempo[0]),
                                                  Integer.parseInt(entradaInicioTempo[1]),
                                                  Integer.parseInt(entradaInicioTempo[2]));

        String[] entradaFim = scanner.nextLine().split(" ");
        int entradaFimDia = Integer.parseInt(entradaFim[1]);
        String[] entradaFimTempo = scanner.nextLine().split(" : ");
        LocalDateTime fim = LocalDateTime.of(2024, 4, entradaFimDia,
                                              Integer.parseInt(entradaFimTempo[0]),
                                              Integer.parseInt(entradaFimTempo[1]),
                                              Integer.parseInt(entradaFimTempo[2]));

        Duration diferenca = Duration.between(inicio, fim);
        long dias = diferenca.toDays();
        long horas = diferenca.toHours() % 24;
        long minutos = diferenca.toMinutes() % 60;
        long segundos = diferenca.getSeconds() % 60;

        System.out.println(dias + " dia(s)");
        System.out.println(horas + " hora(s)");
        System.out.println(minutos + " minuto(s)");
        System.out.println(segundos + " segundo(s)");

        scanner.close();
    }
}

File: Estrutura_Condicional/C#/Ex1061/Main.cs
using System;

namespace Ex1061
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] entradaInicio = Console.ReadLine().Split(' ');
            int entradaInicioDia = int.Parse(entradaInicio[1]);
            string[] entradaInicioTempo = Console.ReadLine().Split(':');
            DateTime inicio = new DateTime(2024, 4, entradaInicioDia,
                                            int.Parse(entradaInicioTempo[0]),
                                            int.Parse(entradaInicioTempo[1]),
                                            int.Parse(entradaInicioTempo[2]));

            string[] entradaFim = Console.ReadLine().Split(' ');
            int entradaFimDia = int.Parse(entradaFim[1]);
            string[] entradaFimTempo = Console.ReadLine().Split(':');
            DateTime fim = new DateTime(2024, 4, entradaFimDia,
                                        int.Parse(entradaFimTempo[0]),
                                        int.Parse(entradaFimTempo[1]),
                                        int.Parse(entradaFimTempo[2]));

            TimeSpan diferenca = fim - inicio;
            long dias = diferenca.Days;
            long horas = diferenca.Hours;
            long minutos = diferenca.Minutes;
            long segundos = diferenca.Seconds;

            Console.WriteLine(dias + " dia(s)");
            Console.WriteLine(horas + " hora(s)");
            Console.WriteLine(minutos + " minuto(s)");
            Console.WriteLine(segundos + " segundo(s)");
        }
    }
}